cmd/generate_example.go

```diff
@@ -19,21 +21,23 @@ func generateExampleRun(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("Couldn't marshal the example to json: %s", err)
 	}
-	err = ioutil.WriteFile(examplePath, exampleJSON, 0600)
+	err = ioutil.WriteFile(outputFileName, exampleJSON, 0600)
+
 	if err != nil {
-		return fmt.Errorf("Couldn't write the example to %s: %s", examplePath, err)
+		return fmt.Errorf("Couldn't write the example to %s: %s", outputFileName, err)
 	}
-	fmt.Printf("Example file created successfully at %s\n", examplePath)
+	fmt.Printf("Example file created successfully at %s\n", outputFileName)
 	return nil
 }
 
 var generateExampleCmd = &cobra.Command{
 	Use:   "example PATH",
```

Comment:
Mention in what way?

Reply:
ping @erbesharat

Reply:
Since we use -o flag for output it's better to mention using flags like the render subcommand instead of using `PATH` in the `Use` field.
@jahzielv As you mentioned in your other comment, Cobra automatically adds `[flags]` to the end of the usage example if the subcommand has any flags.

Reply:
Fixed in latest commit!